dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java

```diff
@@ -0,0 +1,756 @@
+    public static Long waitTimeBeforeJobStatusInTask = TestConfig.tab().longAt(SnappyPrms.jobExecutionTimeInMillisForTask, 6000);
+    public static Long waitTimeBeforeStreamingJobStatusInTask = TestConfig.tab().longAt(SnappyPrms.streamingJobExecutionTimeInMillisForTask, 6000);
+    public static Long waitTimeBeforeJobStatusInCloseTask = TestConfig.tab().longAt(SnappyPrms.jobExecutionTimeInMillisForCloseTask, 6000);
+
+
+    public static <A, B> Map<A, B> toScalaMap(HashMap<A, B> m) {
+        return JavaConverters.mapAsScalaMapConverter(m).asScala().toMap(Predef.<Tuple2<A, B>>conforms());
+    }
+
+    public static void HydraTask_initializeSnappy() {
+        SparkContext sc = null;
+        snc = SnappyContext.getOrCreate(sc);
+        Log.getLogWriter().info("SnappyContext initialized successfully");
+    }
+
+
+    public static void HydraTask_stopSnappy() {
+        snc.stop();
+        Log.getLogWriter().info("SnappyContext stopped successfully");
+    }
+
+    protected String getDataLocation() {
+        String scriptPath = null;
+        try {
+            hd = getHostDescription();
+            scriptPath = hd.getGemFireHome() + sep + ".." + sep + "snappy"
+                    + sep + "quickstart" + sep + "data";
+        } catch (Exception e) {
+            throw e;
+        }
+        return scriptPath;
+    }
+
+    protected HostDescription getHostDescription() {
+        hd = TestConfig.getInstance()
+                .getClientDescription(RemoteTestModule.getMyClientName())
+                .getVmDescription().getHostDescription();
+        sep = hd.getFileSep();
+        return hd;
+    }
+
+    protected String getScriptLocation(String scriptName) {
+        String scriptPath = null;
+        try {
+            hd = getHostDescription();
+            scriptPath = hd.getGemFireHome() + sep + ".." + sep + "snappy"
+                    + sep + "sbin" + sep + scriptName;
+        } catch (Exception e) {
+            throw new TestException("Following error occurred while executing " + e.getMessage());
+        }
+        return scriptPath;
+    }
+
+    protected String getSnappyPath() {
+        hd = getHostDescription();
+        snappyPath = hd.getGemFireHome() + sep + ".." + sep + "snappy";
+        return snappyPath;
+    }
+
+    protected String getParamPath(String paramName) {
+        try {
+            String fullPath = null;
+            if (paramName.contains("/")) {
+                return paramName;
+            } else {
+                fullPath = snappyTest.getDataLocation() + sep + paramName;
+                File path = new File(fullPath);
+                if (path.exists()) {
+                    Log.getLogWriter().info("Data File path found under quickstart data location.");
+                } else {
+                    fullPath = snappyTest.getUserDataLocation(paramName);
+                    Log.getLogWriter().info("Data file path found under dtests data location.");
```

Comment:
should be chaged to ----


if (!path.exists() ) {
fullPath = snappyTest.getUserDataLocation(paramName); 
 if (! (new File(fullPath).exists)  ) 
  throw exception